packages/SwingSet/package.json

```diff
@@ -83,4 +83,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
+}
```

Comment:
The missing terminal newline seems to be a problem in every file in this PR.

Reply:
If this is whitespace policy a requirement I ask that CI fail. We shouldn't have to discuss whitespace on PRs.

I'm happy to make a PR for that, applying Prettier to .json files.

Reply:
> We shouldn't have to discuss whitespace on PRs.

Well, maybe ideally not. But here we are.

I concur; please restore the newlines.

Reply:
I was wrong about `yarn`. If it was yarn doing this I would wait for https://github.com/Agoric/agoric-sdk/pull/5035/ to land before manually changing here because if the tool is going to add newlines then it shouldn't be a manual step to remove them. However it was my IDE settings doing that so 329eed8a4d53f3aa19fe364806add85e5da24980